runtime/lua/nvim/log.lua

```diff
@@ -59,19 +50,15 @@ log.create_functions = function(new_log, new_logger)
 
         local info = debug.getinfo(2, "Sl")
         local log_message = string.format(
-          "%s:%-4s %s",
+          "%s:%s\t%s",
           info.short_src,
           info.currentline,
           message
         )
 
-        if self.levels[name] >= logger.file_level then
+        if self.levels[name] >= logger.log_level then
```

Comment:
I would like to switch the log output condition to 'MIN_LOG_LEVEL'. Is there a way to get 'MIN_LOG_LEVEL'?

Reply:
You mean like [`os.getenv`](https://www.lua.org/pil/22.2.html)?

Reply:
I mean,
https://github.com/neovim/neovim/blob/1852dfdf9aede9f3f8bc670e39c5830c644a9bbc/src/nvim/CMakeLists.txt#L177 , 
https://github.com/neovim/neovim/blob/master/src/nvim/log.c#L121 .
That should be defined on build.

Reply:
Oh, ok, sorry, I know nothing about that then :)

Reply:
I'm not sure we want to use that anyway. It should be possible to enable LSP logging when i e developing a plugin, even with a release build of nvim which doesn't have logging enabled internally.

Reply:
Sure, I'll try it.